package.json

```diff
@@ -29,7 +29,7 @@
     "lodash": "3.10.1",
     "node-ansiparser": "2.0.2",
     "octicons": "2.4.1",
-    "pty.js": "0.3.0",
+    "ptyw.js": "^0.3.2",
```

Comment:
Could you please just change the version to point to your master on Github instead of changing the name?